www/assets/css/educator.scss

```diff
@@ -1,11 +1,6 @@
 #educator-main-section {
-  max-width: $max-content-width;
```

Comment:
Removing this css from here and adding it to .page-single class so it can be reused in multiple pages.